<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultimeterInterface</class>
 <widget class="QMainWindow" name="MultimeterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultimeterInterface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_measure">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить измерение</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Commands</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_result">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>120</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат измерения</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Commands</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="get_status">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить статус</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Commands</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="stop_measure">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить измерение</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Commands</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="set_range">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>60</y>
      <width>193</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Настроить диапазон</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Commands</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>540</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите команду</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>540</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите канал прибора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>540</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите диапазон канала измерения (только для команды &quot;Настроить диапазон&quot;)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>540</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Commands"/>
 </buttongroups>
</ui>
